tests/src/Functional/SubsitePagesTest.php

```diff
@@ -36,6 +36,13 @@ class SubsitePagesTest extends BrowserTestBase {
    */
   protected $adminUser;
 
+  /**
+   * Node to hold the article.
+   *
+   * @var \Drupal\node\NodeInterface
+   */
+  protected $nodeStorage;
```

Comment:
Could use a second opinion here as it doesn't look like this is used so we could even remove this and line https://github.com/localgovdrupal/localgov_subsites/pull/128/files#diff-321651c1769e42606d07218c14d8f37ac09f160358eb1abef5b4c416126931e5R75

Reply:
Yes, please remove this.